src/desktop/apps/auction_support/routes.coffee

```diff
@@ -79,7 +79,7 @@ registerOrRender = (sale, req, res, next) ->
   saleArtwork = new SaleArtwork(artwork: new Artwork(id: req.params.artwork), sale: sale)
   bidderPositions = new BidderPositions(null, { saleArtwork: saleArtwork, sale: sale })
 
-  render = _.after 4, ->
+  render = _.after 3, ->
     res.locals.sd.BIDDER_POSITIONS = bidderPositions.toJSON()
     res.locals.sd.SALE = sale.toJSON()
```

Comment:
Why the change from 3 - 4?

Reply:
Because I removed the `checkRegisteredForAuction` query below (since it got combined into the metaphysics one).